memwatch-next/memwatch-next.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for memwatch-next 0.3.0
+// Project: https://github.com/marcominetti/node-memwatch
+// Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "memwatch-next" {
+    namespace MemWatchNext {
```

Comment:
Instead of declaring a namespace and `export =`, you can just use the body of the namespace as the body of the module.